_data/locales/ja.yml

```
@@ -0,0 +1,31 @@
+ja:
+  locale_name: 日本語
+  nav:
+    about: このサイトについて
+    contribute: 貢献する
+  index:
+    lead: オープンソースソフトウェアはあなたのような人々によって作られています。プロジェクトを立ち上げて大きくしていく方法を学びましょう。
+    opensourcefriday: 今日は金曜日です！少しの時間を、あなたが使っていて愛着を持っているソフトウェアへの貢献に使いましょう。
```

```suggestion
    opensourcefriday: 今日は金曜日です！あなたが使用し愛着を持っているソフトウェアへの貢献に数時間を投資しましょう。
```
言い回しを変えてみました。
ソフトウェアのためという意味を強めるため、Invest を「投資」と変えましたが、
好みで「使う」でも良いと思います。